packages/webapp/.env.example

```diff
@@ -5,4 +5,15 @@ NEXT_PUBLIC_BASESCAN_API_KEY=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 NEXT_PUBLIC_WALLET_CONNECT_PROJECT_ID=XXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
 # Values are 'mainnet' or 'testnet'
-NEXT_PUBLIC_NETWORK_TYPE=testnet
\ No newline at end of file
+NEXT_PUBLIC_NETWORK_TYPE=mainnet
+
+NEXT_PUBLIC_GOLDSKY_BASE_API_URL=https://api.goldsky.com/api/public/project_clz4ukquribdy010b1fgua9nm/subgraphs/gnars-base/latest/gn
+
+# Only using this on SSR for right now, we can make a NEXT_PUBLIC_ envar later one if needed
+PRIVATE_GOLDSKY_L1_SUBGRAPH=https://api.goldsky.com/api/public/project_clwh32mvalyh201vi1ck71seq/subgraphs/gnars-mainnet/latest/gn
+
+# only needed if we are using The Graph, but API usage costs $$ or GRT or you get rate blocked
+PRIVATE_GRAPH_LAYER_1_API_KEY='YOUR PERSONAL API KEY (FOR THE GRAPH NETWORK)'
+
+#TODO: implement and actually make private (ignore for now)
```

Comment:
will make a separate issue for this

Reply:
Relevant to this topic: (reference to my comments on `.env.example` and `env.local.example` below)[https://github.com/gnars-dao/gnars-frontend/pull/48#discussion_r1718785256]